astroid/nodes/node_classes.py

```diff
@@ -2016,13 +2020,17 @@ def postinit(self, nodes: typing.List[NodeNG]) -> None:
         """
         self.nodes = nodes
 
-    def scope(self):
+    def scope(self) -> "LocalsDictNodeNG":
         """The first parent node defining a new scope.
 
         :returns: The first parent scope node.
-        :rtype: Module or FunctionDef or ClassDef or Lambda or GenExpr
+        :rtype: LocalsDictNodeNG
```

Comment:
You could remove it all together. `Sphinx` should pick up the return type annotation. With each CI run, a documentation preview is also generated. You can check it here: https://pylint--1170.org.readthedocs.build/projects/astroid/en/1170/api/astroid.nodes.html#id174